docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  web:
+    build: .
+    command: python /code/forum/manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/code
+    ports:
+      - 8000:8000
+    depends_on:
+      - db
```

Comment:
Use a wait script or the `wait-for-it.sh` or `wait-for-postgres.sh` script to ensure the database is ready before Django starts.
```
command: >
  ./wait-for-it.sh db:5432 -- python /code/forum/manage.py runserver 0.0.0.0:8000
```